src/scripts.js

```diff
@@ -17,6 +17,67 @@ import Sleep from './Sleep';
 
 
 // can i wrap this whole section of new Classes and put that in the api call??
+
+// ########
+let userDataApi, sleepDataApi, actDataApi, hydDataApi, hydrationDataApi, activityDataApi;
+let userApi, sleepApi, actApi, hydApi;
+let getCounter = 0;
+
+function getApiData(ext, callback){
+  fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/" + ext)
+  .then(response => response.json())
+  .then(data => {
+    callback(data);
+    // getCounter++;
```

Comment:
Not sure if you meant to leave these commented out lines in there or not, just wanted to point that out just in case these flew under your radar.